client/src/components/forms/SignUp/index.tsx

```diff
@@ -64,6 +74,57 @@ function SignUpForm({
   setFaculty,
   onNext,
 }: Props) {
+  const [showFirstNameError, setShowFirstNameError] = useState(false);
+  const [showLastNameError, setShowLastNameError] = useState(false);
+  const [showStudentNumberError, setShowStudentNumberError] = useState(false);
+  const [showEmailError, setShowEmailError] = useState(false);
```

Comment:
Is there a reason why we want to maintain these values in state instead of computing them?
E.g.
If we already maintain `firstName` in state, and we can compute whether or not we want to show the error, is there a reason why we would store the error state?

Reply:
Nope there's no reason 😨 Switched it over to computing them in the render